poc/architectures/openrouter.py

```
@@ -0,0 +1,89 @@
+    query: str,
+    context: Optional[str] = None,
+    temperature: float = 0.1,
+    max_tokens: int = 512,
+    timeout_s: float = 30.0,
+) -> dict[str, Any]:
+    """Call OpenRouter chat completion endpoint and return normalized fields."""
+    api_key = os.getenv("OPENROUTER_API_KEY")
+    if not api_key:
+        raise RuntimeError(
+            "OPENROUTER_API_KEY is not set. Export it before running inference."
+        )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)